stable/kong/requirements.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 3.18.3
+  version: 3.9.5
```

Comment:
Not matching with https://github.com/helm/charts/pull/13475/files#diff-509c05d7eb2ab1aaa0709017c4b59a61R3

Reply:
This should be fine since we are pinning 3.9 and any patch version bumps are acceptable.